types/schemas/prototypeApplication/index.ts

```diff
@@ -30,83 +30,301 @@ interface ApplicationSpecification<T extends ApplicationType> {
 
 /**
  * @internal
- * The generic base type for all application types which include a pre-assessment result
- * Child properties may differ based on a granular application type parameter
+ * Some application types will have an extra property 'preAssessment' which contains the PlanX "result"
+ * Ref https://docs.google.com/spreadsheets/d/1FgULPemnwuwysrYGEkReYFXz3n3T7W0nWziU00taZE4/edit?gid=0#gid=0
  */
-interface ApplicationSpecificationWithPreAssessment<T extends ApplicationType> {
```

Comment:
Changed my mind about how to implement `preAssessment` variants - have a tidier solution below that maintains a singular base `Application` spec! 

Eg `export type LawfulDevelopmentCertificateExisting = Application<'ldc.existing'> & PlanXPreAssessment;`